source/assembly_grammar.cpp

```diff
@@ -150,6 +150,7 @@ const SpecConstantOpcodeEntry kOpSpecConstantOpcodes[] = {
     CASE(Select),
     // Comparison
     CASE(IEqual),
+    CASE(INotEqual),
```

Comment:
Please make this a separate change.  It implies a spec change, which is a big thing and needs to be resolved separately.